source/common/upstream/load_balancer_impl.h

```diff
@@ -154,6 +155,8 @@ class LoadBalancerBase : public LoadBalancer {
   DegradedAvailability per_priority_degraded_;
   // Levels which are in panic
   std::vector<bool> per_priority_panic_;
+  // The total count of healthy hosts across all priority levels.
+  uint32_t total_healthy_hosts_;
```

Comment:
I think it would be better to revert all of this also as unrelated. If not reverted, does this need to be zero initialized somewhere?